planetiler-core/src/main/java/com/onthegomap/planetiler/reader/GeoPackageReader.java

```diff
@@ -138,7 +149,20 @@ public void readFeatures(Consumer<SimpleFeature> next) throws Exception {
       MathTransform transform = (coordinateTransform != null) ? coordinateTransform :
         CRS.findMathTransform(CRS.decode("EPSG:" + srsId), latLonCRS);
 
-      for (var feature : features.queryForAll()) {
+      FeatureIndexManager indexer = new FeatureIndexManager(geoPackage,
+        features);
+      indexer.setIndexLocation(FeatureIndexType.RTREE);
```

Comment:
It's been a while, but if I remember correctly, this will actually build the index if it doesn't already exist in the DB before running the query.

Looking at the PR where I initially added this, should be possible to check for the presence of the index before trying to use it: https://github.com/onthegomap/planetiler/pull/413#discussion_r1051006021